cmd/qat_plugin/README.md

```diff
@@ -53,6 +53,7 @@ The QAT plugin can take a number of command line arguments, summarised in the fo
 | -kernel-vf-drivers | string | Comma separated VF Device Driver of the QuickAssist Devices in the system. Devices supported: DH895xCC, C62x, C3xxx, 4xxx, C4xxx and D15xx (default: `c6xxvf,4xxxvf`) |
 | -max-num-devices | int | maximum number of QAT devices to be provided to the QuickAssist device plugin (default: `32`) |
 | -mode | string | plugin mode which can be either `dpdk` or `kernel` (default: `dpdk`) |
+| -allocation-policy | string | none | 3 possible values: balanced, packed, none. Balanced mode spreads allocated QAT VF resources balanced among QAT PF devices, and packed mode packs one QAT PF device full of QAT VF resources before allocating resources from the next QAT PF. (default: `none`) |
```

Comment:
Please document also what "none" does. In "none" is really random, I guess it means that all the physical endpoints will be probabilistically balanced? Would it make sense to drop "none" policy altogether and just go with "balanced" and "packed", with "packed" being the default?

Reply:
valid comment about the need to have `none` at all. For GPUs it was originally called 'random' but the functionality was not randomizing anything but simply taking the first X deviceIDs from the allocation request. I'm OK going with the two "opt-in" policies without `none`

Reply:
Ok, then let me change like that.!

Reply:
Ok. Now that I think more about it, maybe `balanced` might actually be a better default. What do you think?

Reply:
> Ah, balanced as default? To me too, it makes sense to have balanced as default. @mythi What do you think?

we want it so that when no parameter is set, there's no policy and that should be the default.

Reply:
So, could you let me know what is the decision?
IMO, "no policy" is not possible to implement. There has to be a default policy (`none` policy as default if neither `balanced` nor `packed` can be default)

Reply:
@ipuustin There is no explanation for `none` in GPU plugin's case. Wdyt?

Reply:
IMO, if the default policy should be 'no policy', then having just `none` policy as before would be the best implementation.!

Reply:
Since we add
`// Implement the PreferredAllocator interface.`
the case where user has not specified `-allocation-policy` must be handled. It does not mean we have to accept `-allocation-policy none`.

Reply:
1. There is 'none' option. Only two options "balanced" and "packed" are provided.
2. There is no default option. When no policy is specified, then we use the function "nonePolicy", however it does not mean that a user can input any option to use that function.